modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionOverview/components/CardSubscription/CardSubscription.js

```diff
@@ -11,13 +11,13 @@
 
 import {useModal} from '@clayui/modal';
 import {useState} from 'react';
-import i18n from '../../../../common/I18n';
-import {StatusTag} from '../../../../common/components';
-import {useAppPropertiesContext} from '../../../../common/contexts/AppPropertiesContext';
-import {SLA_STATUS_TYPES} from '../../../../common/utils/constants';
-import getDateCustomFormat from '../../../../common/utils/getDateCustomFormat';
-import getKebabCase from '../../../../common/utils/getKebabCase';
-import ModalCardSubscription from '../../containers/ModalCardSubscription';
+import i18n from '../../../../../../../../../common/I18n';
+import {StatusTag} from '../../../../../../../../../common/components';
+import {useAppPropertiesContext} from '../../../../../../../../../common/contexts/AppPropertiesContext';
+import {SLA_STATUS_TYPES} from '../../../../../../../../../common/utils/constants';
+import getDateCustomFormat from '../../../../../../../../../common/utils/getDateCustomFormat';
+import getKebabCase from '../../../../../../../../../common/utils/getKebabCase';
+import ModalCardSubscription from '../../../../../../../containers/ModalCardSubscription';
 
```

Comment:
Hey @joaodamasceno2001 I think ModalSubscription should be a component of the Card Subscription. What do you think? This component is outside the SubscriptionOverview?